### PR TITLE
Add controller and interpreter

### PR DESCRIPTION
**Pull request by stasyukhymenko**

### CODE REVIEW

File: src/main/java/org/disguys/brainfuckbackend/service/BrainfuckController.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.disguys.brainfuckbackend.service;
+
+import java.util.List;
+
+public class BrainfuckController {
+    public String processBrainfuck(String code, String userInput) {
+        System.out.println("Start processing");
```
Comment (thent1) [inline src/main/java/org/disguys/brainfuckbackend/service/BrainfuckController.java:7]: Remove logging

File: src/main/java/org/disguys/brainfuckbackend/service/BrainfuckInterpreter.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    public String interpret(BrainfuckParser.ASTNode root) {
+        System.out.println("Intepreting");
+        byte[] memory = new byte[MEMORY_SIZE];
+        int pointer = 0;
+        interpretNode(root, memory, pointer);
+        return output.toString();
+    }
+
+    private void interpretNode(BrainfuckParser.ASTNode node, byte[] memory, int pointer) {
+        System.out.println("Intepreting node");
```
Comment (thent1) [inline src/main/java/org/disguys/brainfuckbackend/service/BrainfuckInterpreter.java:27]: Remove logging

File: src/main/java/org/disguys/brainfuckbackend/service/BrainfuckInterpreter.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    private boolean inloopFlag;
+
+    public BrainfuckInterpreter(String input) {
+        this.input = input;
+        this.inputPointer = 0;
+        this.output = new StringBuilder();
+    }
+
+    public String interpret(BrainfuckParser.ASTNode root) {
+        System.out.println("Intepreting");
```
Comment (GIroSTarK) [inline src/main/java/org/disguys/brainfuckbackend/service/BrainfuckInterpreter.java:19]: Remove loggings

**Review by thent1 — CHANGES_REQUESTED** [review]

File: src/main/java/org/disguys/brainfuckbackend/service/BrainfuckInterpreter.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                        memory[pointer] = 0;
+                    }
+                    break;
+                case LOOP_START:
+                    while (memory[pointer] != 0) {
+                        interpretNode(child, memory, pointer);
+                    }
+                    break;
+                case LOOP_END:
+
```
Comment (GIroSTarK) [inline src/main/java/org/disguys/brainfuckbackend/service/BrainfuckInterpreter.java:65]: Useless empty line

**Review by GIroSTarK — CHANGES_REQUESTED** [review]

**Review by GIroSTarK — APPROVED** [review]
Good job!😘

**Pull request merged**